### ISSUE TITLE
[Help] 全参数微调，loss几乎不下降，loss为6.x

### ISSUE THREAD

**Issue by liuzhidanhhh** [THUDM/ChatGLM-6B#1213]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

用教程中的命令bash ds_train_finetune.sh跑全量。--num_gpus=8，其余参数默认。loss不下降。
部分loss如下：

***** train metrics *****
  epoch                    =        1.4
  train_loss               =      6.734
  train_runtime            = 0:48:17.70
  train_samples            =     114599
  train_samples_per_second =     55.216
  train_steps_per_second   =      1.726

2023-06-08 19:00:45,652] [INFO] [timer.py:199:stop] epoch=0/micro_step=4960/global_step=4960, RunningAvgSamplesPerSec=62.9432947371848, CurrSamplesPerSec=63.18144236137459, MemAllocated=21.59GB, MaxMemAllocated=28.8GB

 99%|█████████▉| 4960/5000 [46:37<00:20,  1.96it/s]
                                                   
{'loss': 6.6606, 'learning_rate': 9.6e-07, 'epoch': 1.38}

 99%|█████████▉| 4960/5000 [46:37<00:20,  1.96it/s]
 99%|█████████▉| 4961/5000 [46:38<00:19,  1.96it/s]
 99%|█████████▉| 4962/5000 [46:38<00:19,  1.96it/s]
 99%|█████████▉| 4963/5000 [46:39<00:18,  1.96it/s]
 99%|█████████▉| 4964/5000 [46:39<00:18,  1.96it/s]
 99%|█████████▉| 4965/5000 [46:40<00:17,  1.96it/s]
 99%|█████████▉| 4966/5000 [46:40<00:17,  1.96it/s]
 99%|█████████▉| 4967/5000 [46:41<00:16,  1.96it/s]
 99%|█████████▉| 4968/5000 [46:41<00:16,  1.96it/s]
 99%|█████████▉| 4969/5000 [46:42<00:15,  1.96it/s][2023-06-08 19:00:50,580] [INFO] [logging.py:96:log_dist] [Rank 0] step=4970, skipped=8, lr=[7.6e-07, 7.6e-07], mom=[(0.9, 0.999), (0.9, 0.999)]
[2023-06-08 19:00:50,749] [INFO] [timer.py:199:stop] epoch=0/micro_step=4970/global_step=4970, RunningAvgSamplesPerSec=62.94369776396332, CurrSamplesPerSec=63.16110542746703, MemAllocated=21.59GB, MaxMemAllocated=28.8GB

 99%|█████████▉| 4970/5000 [46:42<00:15,  1.96it/s]
                                                   
{'loss': 6.6333, 'learning_rate': 7.6e-07, 'epoch': 1.39}

 99%|█████████▉| 4970/5000 [46:42<00:15,  1.96it/s]
 99%|█████████▉| 4971/5000 [46:43<00:14,  1.96it/s]
 99%|█████████▉| 4972/5000 [46:43<00:14,  1.96it/s]
 99%|█████████▉| 4973/5000 [46:44<00:13,  1.96it/s]
 99%|█████████▉| 4974/5000 [46:44<00:13,  1.96it/s]
100%|█████████▉| 4975/5000 [46:45<00:12,  1.96it/s]
100%|█████████▉| 4976/5000 [46:45<00:12,  1.96it/s]
100%|█████████▉| 4977/5000 [46:46<00:11,  1.96it/s]
100%|█████████▉| 4978/5000 [46:47<00:11,  1.96it/s]
100%|█████████▉| 4979/5000 [46:47<00:10,  1.96it/s][2023-06-08 19:00:55,676] [INFO] [logging.py:96:log_dist] [Rank 0] step=4980, skipped=8, lr=[5.6e-07, 5.6e-07], mom=[(0.9, 0.999), (0.9, 0.999)]
[2023-06-08 19:00:55,844] [INFO] [timer.py:199:stop] epoch=0/micro_step=4980/global_step=4980, RunningAvgSamplesPerSec=62.94408793953609, CurrSamplesPerSec=63.25880651771861, MemAllocated=21.59GB, MaxMemAllocated=28.8GB

100%|█████████▉| 4980/5000 [46:48<00:10,  1.96it/s]
                                                   
{'loss': 6.6173, 'learning_rate': 5.6e-07, 'epoch': 1.39}

100%|█████████▉| 4980/5000 [46:48<00:10,  1.96it/s]
100%|█████████▉| 4981/5000 [46:48<00:09,  1.96it/s]
100%|█████████▉| 4982/5000 [46:49<00:09,  1.96it/s]
100%|█████████▉| 4983/5000 [46:49<00:08,  1.96it/s]
100%|█████████▉| 4984/5000 [46:50<00:08,  1.96it/s]
100%|█████████▉| 4985/5000 [46:50<00:07,  1.96it/s]
100%|█████████▉| 4986/5000 [46:51<00:07,  1.96it/s]
100%|█████████▉| 4987/5000 [46:51<00:06,  1.96it/s]
100%|█████████▉| 4988/5000 [46:52<00:06,  1.96it/s]
100%|█████████▉| 4989/5000 [46:52<00:05,  1.96it/s][2023-06-08 19:01:00,771] [INFO] [logging.py:96:log_dist] [Rank 0] step=4990, skipped=8, lr=[3.6e-07, 3.6e-07], mom=[(0.9, 0.999), (0.9, 0.999)]
[2023-06-08 19:01:00,940] [INFO] [timer.py:199:stop] epoch=0/micro_step=4990/global_step=4990, RunningAvgSamplesPerSec=62.94446569583285, CurrSamplesPerSec=63.20325072318608, MemAllocated=21.59GB, MaxMemAllocated=28.8GB

100%|█████████▉| 4990/5000 [46:53<00:05,  1.96it/s]
                                                   
{'loss': 6.6624, 'learning_rate': 3.6e-07, 'epoch': 1.39}

100%|█████████▉| 4990/5000 [46:53<00:05,  1.96it/s]
100%|█████████▉| 4991/5000 [46:53<00:04,  1.96it/s]
100%|█████████▉| 4992/5000 [46:54<00:04,  1.96it/s]
100%|█████████▉| 4993/5000 [46:54<00:03,  1.96it/s]
100%|█████████▉| 4994/5000 [46:55<00:03,  1.96it/s]
100%|█████████▉| 4995/5000 [46:55<00:02,  1.96it/s]
100%|█████████▉| 4996/5000 [46:56<00:02,  1.96it/s]
100%|█████████▉| 4997/5000 [46:56<00:01,  1.96it/s]
100%|█████████▉| 4998/5000 [46:57<00:01,  1.96it/s]
100%|█████████▉| 4999/5000 [46:57<00:00,  1.96it/s][2023-06-08 19:01:05,875] [INFO] [logging.py:96:log_dist] [Rank 0] step=5000, skipped=8, lr=[1.6e-07, 1.6e-07], mom=[(0.9, 0.999), (0.9, 0.999)]
[2023-06-08 19:01:06,044] [INFO] [timer.py:199:stop] epoch=0/micro_step=5000/global_step=5000, RunningAvgSamplesPerSec=62.944661662179186, CurrSamplesPerSec=62.468573428578345, MemAllocated=21.59GB, MaxMemAllocated=28.8GB

100%|██████████| 5000/5000 [46:58<00:00,  1.96it/s]
                                                   
{'loss': 6.6543, 'learning_rate': 1.6e-07, 'epoch': 1.4}

### Expected Behavior

loss正常下降，readme中为3.x

### Steps To Reproduce

1. --num_gpus=8，其余参数默认。
2. bash ds_train_finetune.sh

### Environment

```markdown
- OS:ubuntu
- Python:2.7
- Transformers:
- PyTorch:
- CUDA Support (`python -c "import torch; print(torch.cuda.is_available())"`) :
```

**Comment by hailuoS** [comment]
一样的问题，请问你解决了吗，我也是全量微调loss6.几就下不去了，看官方readme里应该能到降到3

**Comment by hailuoS** [comment]
可以尝试下把学习率改成1e-5，我这边修改后训练loss是可以下降到3.x，而且运行评估脚本计算出来的指标也差不多对得上

**Comment by zzy347964399** [comment]
> 可以尝试下把学习率改成1e-5，我这边修改后训练loss是可以下降到3.x，而且运行评估脚本计算出来的指标也差不多对得上

@hailuoS 不好意思请问一下为什么学习率减小，反而loss下降更快呀？
